library/gcm.c

```diff
@@ -306,7 +274,7 @@ static inline void gcm_mult_largetable(uint64_t m[256][2], const uint8_t *x, uin
     u64z[1] = 0;
 
     for (i = 15; i > 0; i--) {
-        mbedtls_xor16(u8z, u8z, m[x[i]]);
+        mbedtls_xor(u8z, u8z, (unsigned char *) m[x[i]], 16);
```

Comment:
Please use `mbedtls_xor_no_simd` (here and elsewhere)

Reply:
Done